group-prs.sh

```
@@ -0,0 +1,64 @@
+# Input JSON file (passed as a parameter)
+INPUT_JSON="$1"
+PLUGINS_JSON="$2"
+
+# Validate that the input JSON file exists and is readable
+if [ ! -r "$INPUT_JSON" ]; then
+  echo "Error: Input file '$INPUT_JSON' does not exist or is not readable" >&2
+  exit 1
+fi
+
+# Check if the plugins JSON file exists and is less than one day old
+if [ ! -f "$PLUGINS_JSON" ] || [ $(find "$PLUGINS_JSON" -mtime +1 -print) ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)